lib/config/presets/internal/replacements.ts

```diff
@@ -51,6 +52,18 @@ export const presets: Record<string, Preset> = {
     ],
     ignoreDeps: [], // Hack to improve onboarding PR description
   },
+  'airbnb-prop-types-to-prop-types-tools': {
+    '`airbnb-prop-types` was given to a new maintainer and renamed',
```

Comment:
I was kinda expecting a test to shout at us for missing the `description:` field. Should we test for this?

Reply:
@rarkins do you think we should test for missing `description` fields in our presets?

The docs use the `description` field to display a short summary of the preset. So it's best if it's always filled in.

Reply:
For replacements perhaps

Reply:
Do you want me to create an issue to check the `description` for _replacement_ presets only?